polaris/benchmark/_base.py

```diff
@@ -18,7 +18,7 @@
 from sklearn.utils.multiclass import type_of_target
 
 from polaris._artifact import BaseArtifactModel
-from polaris._mixins import ChecksumMixin
+from polaris.mixins._checksum import ChecksumMixin
```

Comment:
To maintain consistency with the current codebase, we could create a `polaris/mixins/__init__.py` file with the following content:

```python
# Import classes from the internal modules
from polaris.mixins._checksum import ChecksumMixin
from polaris.mixins._format_text import FormattingMixin

# Define what is available for import when the package is imported
__all__ = ["ChecksumMixin", "FormattingMixin"]
```

This will allow us to do:

```python
from polaris.mixins import ChecksumMixin
from polaris.mixins import FormattingMixin
```

Reply:
Good call, thank you!